pmd-core/src/main/java/net/sourceforge/pmd/cli/internal/ExecutionResult.java

```diff
@@ -0,0 +1,17 @@
+package net.sourceforge.pmd.cli.internal;
+
+public enum ExecutionResult {
```

Comment:
Not sure if this was duplicated on purpose, but there is a similar enum in the class PMD: https://docs.pmd-code.org/apidocs/pmd-core/6.47.0/net/sourceforge/pmd/PMD.StatusCode.html

Reply:
Yes, and we also have `CPD.StatusCode` (which is separate, but uses the same values). My intention is to remove both and unify everything. As we are having a single main, a single CLI, I wanted to ensure we have a single point where to manage status codes. In my local branch, I extended the `pmd -h` command to display exit codes along all subcommands.